helix-term/src/commands.rs

```diff
@@ -4828,33 +4828,35 @@ fn transpose_view(cx: &mut Context) {
     cx.editor.transpose_view()
 }
 
-// split helper, clear it later
-fn split(cx: &mut Context, action: Action) {
-    let (view, doc) = current!(cx.editor);
+/// Open a new split in the given direction specified by the action.
+///
+/// Maintain the current view (both the cursor's position and view in document).
+fn split_helper(editor: &mut Editor, action: Action) {
```

Comment:
Avoid renames, they pollute diffs and and in RBIs case I also preferred the old naming. Similailry the argument should stay a context (same as for all.functions in commands.rs)

Reply:
I thought this too when I was initially review this pr but the context for type-able commands and regular commands is different so the editor needs to be passed in instead of context so they can all use the same helper function

Reply:
Let me know if there's a way to maybe convert between the two context types

Reply:
I don't know if something like this could be used: 

https://github.com/helix-editor/helix/blob/48373d4a2b6402dce91ebcdf29b6341fcf0ba302/helix-term/src/commands.rs#L2838-L2845

Reply:
Ah sorry for not replying, if we only need the editor here it's better to just pass that instead of constructing a new context.

Reply:
yeah I missed that in that case changing the argument is fine I suppose but the function still shouldnt' be renamed